packages/react-select/src/Select.js

```diff
@@ -1767,14 +1767,12 @@ export default class Select extends Component<Props, State> {
       } else {
         const input =
           selectValue.length > 0 ? (
-            selectValue.map((opt, i) => (
-              <input
-                key={`i-${i}`}
-                name={name}
-                type="hidden"
-                value={this.getOptionValue(opt)}
-              />
-            ))
+            selectValue.map(opt => {
+              const value = this.getOptionValue(opt);
```

Comment:
This value might not be unique at all for `key`. 


Make this customizable by user instead by some callback, like `this.getReactKey(opt,i)`, because I as a user know how to specify unique id for my data.

Reply:
@Rall3n Is "value" of option required to be unique or not  ? I assume it should be ... but I found no information about that in docs.

Reply:
@NullPainter2 it's likely assumed, but not sure that it's a stated requirement. Perhaps something to consider. I wonder if something like `${label}_${value}` would be acceptable as I can't think of a compelling reason why or how a user would have duplicate derived value and label values. Perhaps in different option groups, but they would have a different parent node so the keys I believe wouldn't need to be unique.